### PR TITLE
Feature/lattice 2426 scheduled task service

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

**Review by makosblade — APPROVED** [review]

**Review by geekbeast — APPROVED** [review]
Good enough for v1 :)

**Review by dbt — APPROVED** [review]

File: src/main/kotlin/com/openlattice/codex/CodexService.kt [hunk 1]
```diff
@@ -252,9 +275,37 @@ class CodexService(
         return EntityDataKey(entitySetId, entityKeyId)
     }
 
+    fun getScheduledMessagesForOrganization(organizationId: UUID): Map<UUID, MessageRequest> {
+        return BasePostgresIterable(PreparedStatementHolderSupplier(hds, GET_SCHEDULED_MESSAGES_FOR_ORG_SQL) {
+            it.setString(1, organizationId.toString())
+        }) {
+            val task = ResultSetAdapters.schedulableTask(it).task as SendCodexMessageTask
```
Comment (dbt) [inline src/main/kotlin/com/openlattice/codex/CodexService.kt]: This is now `scheduledTask` in your c-c PR.

File: src/main/kotlin/com/openlattice/codex/CodexService.kt [hunk 2]
```diff
@@ -252,9 +275,37 @@ class CodexService(
...
+            ResultSetAdapters.id(it) to task.message
+        }.toMap()
+    }
+
+    fun getScheduledMessagesForOrganizationAndPhoneNumber(organizationId: UUID, phoneNumber: String): Map<UUID, MessageRequest> {
+        return BasePostgresIterable(PreparedStatementHolderSupplier(hds, GET_SCHEDULED_MESSAGES_FOR_ORG_AND_PHONE_SQL) {
+            it.setString(1, organizationId.toString())
+            it.setString(2, DataTables.quote(phoneNumber))
+        }) {
+            val task = ResultSetAdapters.schedulableTask(it).task as SendCodexMessageTask
```
Comment (dbt) [inline src/main/kotlin/com/openlattice/codex/CodexService.kt]: See above.

**Review by dbt — COMMENTED** [review]
code still lgtm but one thing to correct before merging.

File: src/main/kotlin/com/openlattice/codex/CodexService.kt (hunk 1 above)
Comment (katherinebernstein) [reply src/main/kotlin/com/openlattice/codex/CodexService.kt]: ooh yes thanks!!

**Pull request merged**